pkg/middleware/recovery.go

```diff
@@ -52,7 +53,7 @@ func stack(skip int) []byte {
 		// Print this much at least.  If we can't find the source, it won't show.
 		fmt.Fprintf(buf, "%s:%d (0x%x)\n", file, line, pc)
 		if file != lastFile {
-			data, err := ioutil.ReadFile(file)
+			data, err := ioutil.ReadFile(filepath.Clean(file))
```

Comment:
After reviewing the gosec [issue thread](https://github.com/securego/gosec/issues/439), I think we should silence this warning, since `filepath.Clean` doesn't resolve any security issue.